WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/TitleAndValueRow.swift

```diff
@@ -18,6 +18,8 @@ struct TitleAndValueRow: View {
                 AdaptiveStack(horizontalAlignment: .leading) {
                     Text(title)
                         .style(bold: bold)
+                        .multilineTextAlignment(.leading)
```

Comment:
> Another minor issue that I found: when the tax line title is a bit long, the text seems to be center aligned, hence some extra space on the left of the text. Do you think making it align to the left will make it looks nicer?

I thought of having a property `titleAlignment` (similar to`bold`) with `.center` as the default value  and passing in `.leading` from `PaymentsSection` View. But, `.center` isn't really the default value for `Text`. I think the text is aligned `center` due to the alignment logic in `AdaptiveStack`.

Also, I think that using `.leading` as the value for `.multilineTextAlignment` will suit all places `TitleAndValueRow` is used from.

 I am not proficient in SwiftUI so kindly let me know if you think that this isn't a good idea.

Reply:
> I think the text is aligned center due to the alignment logic in AdaptiveStack.

This seems to be correct! In the case where I found the issue, I was using a device with compact horizontal size class, so `AdaptiveStack` uses an HStack and we cannot really control the horizontal alignment in the stack directly. 

So, using `multilineTextAlignment` is the most sensible approach in my opinion. I'm wondering if we should use the same modifier on the value text as well, but with `.trailing` alignment to make sure the text looks good if it has more than one line.

Also, I'd suggest adding an extra view for the previews of the `TitleAndValueRow` view - to demonstrate the multi-line cases. WDYT?